opts/opts.go

```diff
@@ -17,6 +17,7 @@ var (
 	alphaRegexp       = regexp.MustCompile(`[a-zA-Z]`)
 	domainRegexp      = regexp.MustCompile(`^(:?(:?[a-zA-Z0-9]|(:?[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9]))(:?\.(:?[a-zA-Z0-9]|(:?[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])))*)\.?\s*$`)
 	DefaultHTTPHost   = "127.0.0.1"            // Default HTTP Host used if only port is provided to -H flag e.g. docker -d -H tcp://:8080
+	DefaultHTTPPort   = 2375                   // Default HTTP Port
```

Comment:
2375 is for HTTPS, 2376 is for HTTP. I would love to make HTTPS by default but i'm not sure how.